### PR TITLE
Channel/rubocop 0.70

### PR DESCRIPTION
**Pull request by marcelotoledo5000**
Upgrades:

* Rubocop version to 0.70.0

### CODE REVIEW

**Comment by marcelotoledo5000** [comment]
I still have a problem with this test... 

https://circleci.com/gh/codeclimate/codeclimate-rubocop/456?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

**Comment by marcelotoledo5000** [comment]
CC: @ale7714 @brynary

File: Gemfile [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-gem "activesupport", require: false
-gem "mry", "~> 0.52.0", require: false
-gem "parser"
-gem "pry", require: false
-gem "rubocop", "~> 0.67.0", require: false
-gem "rubocop-migrations", require: false
-gem "rubocop-performance", require: false
-gem "rubocop-rspec", require: false
-gem "safe_yaml"
+gem 'activesupport', require: false
```
Comment (filipesperandio) [inline Gemfile]: Although not enforced in this repo, we tend to prefer double quotes.

Comment (marcelotoledo5000) [reply Gemfile]: No problem, I changed again.

**Comment by marcelotoledo5000** [comment]
> We created the `channel/rubocop-0-70` using the `bin/newver` script yesterday.
> This PRs still have a few interesting things, do you mind rebasing off that branch?

Nice, I did!

File: Gemfile [hunk 2]
```diff
@@ -6,9 +6,8 @@ gem "activesupport", require: false
 gem "mry", "~> 0.52.0", require: false
 gem "parser"
 gem "pry", require: false
-gem "rubocop", "~> 0.67.0", require: false
+gem "rubocop", "~> 0.70.0", require: false
 gem "rubocop-migrations", require: false
-gem "rubocop-performance", require: false
```
Comment (granth) [inline Gemfile]: Was it intentional to remove `rubocop-performance`? See #187.

**Comment by marcelotoledo5000** [comment]
I'm so sorry, I think that was happened when I did the rebase... I can put again...

**Comment by marcelotoledo5000** [comment]
Nice, make sense... And your suggest I should open new PRs to same branch or the other, master?

**Comment by filipesperandio** [comment]
@marcelotoledo5000 Do it against `master`. Not 100% sure about the ruby upgrade, but we will figure.

**Comment by ale7714** [comment]
Thank you @marcelotoledo5000 

@filipesperandio let's go with the Ruby upgrade too. Given that ruby 2.3 is unsupported.

**Comment by marcelotoledo5000** [comment]
> Sorry for the back and forth but do you mind splitting this up?
> There are multiple concerns in this PR I think it is valid to get them in individually.
> 
>     * I see there is a change on the way we import the docs, which seems like a valid thing, so I would put all related changes in their own PR.
> 
>     * Another one with the typos/copy changes you are suggesting to `README.md` and `DEVELOPERS.md`
> 
>     * A 3rd one with Dockerfile updates

@filipesperandio,

I think it would be best if you accept the following pull requests, before finalizing this one:

- https://github.com/codeclimate/codeclimate-rubocop/pull/197 -> Upgrades Ruby and Dockerfile -> into channel/rubocop-0-67 or to 68;
- https://github.com/codeclimate/codeclimate-rubocop/pull/168 -> Rubocop 0.68
- https://github.com/codeclimate/codeclimate-rubocop/pull/171 -> Rubocop 0.69
- And then, I'll resolve all conflicts here and this one will can be done.

What do you think?

**Comment by filipesperandio** [comment]
@marcelotoledo5000 
#168 and #171 state they add rubocop 0.68 and 0.69 respectively. We did that through our scripts already and those channels are now available. If they bring something else, let's rebase them off the right channel branches and reword the intent.

I was expecting #198 to be small (basically DEVELOPERS and READ files) and against `master`, which we could later bring down to the channels - if ever - since they are basically read by the majority through the master branch.

#197 makes sense, will look into it.

Would there be anything left here?

**Pull request closed without merging**